weeder.cabal

```diff
@@ -78,6 +78,8 @@ test-suite weeder-test
   type: exitcode-stdio-1.0
   main-is: Spec.hs
   hs-source-dirs: test
+  build-tool-depends:
+    hspec-discover:hspec-discover
```

Comment:
This was necessary to get nix to put `hspec-discover` on the path for `cabal test` to run.